test/node/msw-api/setup-server/input-validation.node.test.ts

```diff
@@ -14,6 +14,6 @@ test('throws an error given an Array of request handlers to setupServer', () =>
   }
 
   expect(createServer).toThrow(
-    `[MSW] Failed to construct "SetupServerApi" given an Array of request handlers. Make sure you spread the request handlers when calling the respective setup function.`,
+    `[MSW] Failed to construct "SetupNodeServerApi" given an Array of request handlers. Make sure you spread the request handlers when calling the respective setup function.`,
```

Comment:
I don't like the subclass name leaking to the consumer. We should somehow keep this as `SetupServerApi`.

Reply:
Will fix this by not relying on the constructor name in error messages and instead introducing a new `displayName` property on the setup class.